src/master-playlist-controller.js

```diff
@@ -889,17 +895,29 @@ export class MasterPlaylistController extends videojs.EventTarget {
       // it, instead of erring the player or retrying this playlist, clear out the current
       // blacklist. This allows other playlists to be attempted in case any have been
       // fixed.
-      videojs.log.warn('Removing all playlists from the blacklist because the last ' +
-                       'rendition is about to be blacklisted.');
+      let unblacklisted = false;
+
       playlists.forEach((playlist) => {
-        if (playlist.excludeUntil !== Infinity) {
+        // skip current playlist which is about to be blacklisted
+        if (playlist === currentPlaylist) {
```

Comment:
The current playlist is about to be blacklisted, so it should not cause us to log that **other** playlists are going to be unblacklisted.